pagopa/io-app
# chore(IT Wallet): [SIW-1379] Add informational alerts in credential details screens

## Short description
This PR adds the credential card status and informational alerts in the credential details screen

## List of changes proposed in this pull request
- Added `getCredentialExpireDate` utility function to get the credential expiration date from its claims, if present
- Added `getCredentialExpireDays` utility function to get the credential remaining days until its expiration
- Added `getCredentialExpireStatus` utility function to get the credential expiration status
- Added alerts in credential details screen component
- Added credential card status visualization in credential details screen component
- Removed grouping for credential claims in credential details screen

## How to test
Open the MDL credential screen details, check that the alert is correctly displayed and the bottom sheet is presented by tapping on it.
Additionally, you should be able to get other alerts (expired, expiring) by mocking the expiration date of the credential.

## Previews

<img src="https://github.com/user-attachments/assets/d5f5c3ae-d8c2-4992-bfde-3d1d28424c05" width="200" />
<img src="https://github.com/user-attachments/assets/cd8adf2f-8534-4e98-9bf0-0f531b5a5524" width="200" />
<img src="https://github.com/user-attachments/assets/c1570617-8851-423d-8ece-39160c40cb29" width="200" />
